.github/ISSUE_TEMPLATE.md

```diff
@@ -1,22 +0,0 @@
-<!-- Please don't delete this template or we'll close your issue -->
```

Comment:
Why did you remove these files?

Reply:
He is still learning git^^

Reply:
LOL @sokra ever since these got changed, I've had line ending warnings and these get thrown into every PR. I'm not sure why this keeps happening so yes I'm still learning git :joy: